package.json

```diff
@@ -7,8 +7,7 @@
     "addons-linter": "bin/addons-linter"
   },
   "scripts": {
-    "start": "grunt start",
```

Comment:
Given that we use `jest` these days this isn't really needed anymore. At least, I haven't used it since I joined the linter project so I don't see why it should stay.

Reply:
Seems like this command currently just does `webpack --watch ...`, or am I missing anything? Could probably be replaced with `npm run build -- --watch`...

Reply:
Kinda, but even that isn't really needed since ages now. We use jest which transpiles / builds the modules dynamically for us.